lib/elsa/dynamic_process_manager.ex

```diff
@@ -23,7 +23,7 @@ defmodule Elsa.DynamicProcessManager do
     }
   end
 
-  def ready?(server, timeout \\ 10_000) do
+  def wait_ready!(server, timeout \\ 10_000) do
```

Comment:
DynamicProcessorModule is documented with @moduledoc false, so I believe I am within my rights to just change the function name entirely.